test/kotlin/integration/StreamIdleTimeoutTest.kt

```diff
@@ -93,7 +93,7 @@ static_resources:
       - lb_endpoints:
         - endpoint:
             address:
-              socket_address: { address: 127.0.0.1, port_value: 10101 }
+              socket_address: { address: 127.0.0.1, port_value: $remotePort }
```

Comment:
Would it make sense to apply this randomization for our `TestEngineBuilder` classes as well to help downstream consumers avoid this problem?

Reply:
Sure, I can see it being useful. I'll update them in another PR.

Reply:
great, thanks